### PR TITLE
[VTK9.2.6] const var, vtk_include_dirs and LF file

### PR DESCRIPTION
**Pull request by mathildemerle**
This PR is needed for https://github.com/medInria/medInria-public/pull/1133 and same PR on MUSICardio.
This is needed to compile VTK 9.2.6.

[itkFiberBundleStatisticsCalculator.cxx](https://github.com/medInria/TTK/compare/master...mathildemerle:TTK:ITK5+VTK9#diff-b62a99c7218b620c8bfde011a6adc3d5a1eeafdce939f072e812da5ee73edca6) changes are due to a LF/CRLF change, but the only real change is line 52.

After this merge we need to create a ITK5.4rc04+VTK9 branch.

:m:

### CODE REVIEW

File: Algorithms/itkFiberBundleStatisticsCalculator.cxx [hunk 1]
```diff
@@ -1,257 +1,257 @@
...
+            return;
+        }
+
+        vtkDataArray *tCoefficients = m_Input->GetPointData()->GetArray("Tensors");
+        vtkPoints* points = m_Input->GetPoints();
+
+        lines->InitTraversal();
+  
+        vtkIdType npts  = 0;
+        vtkIdType const* ptids = nullptr;
```
Comment (mathildemerle) [inline Algorithms/itkFiberBundleStatisticsCalculator.cxx:52]: only changed line here

Comment (fcollot) [reply Algorithms/itkFiberBundleStatisticsCalculator.cxx:52]: why are all the other lines appearing as modified? (this isn't ideal for the git history)

Comment (mathildemerle) [reply Algorithms/itkFiberBundleStatisticsCalculator.cxx:52]: This file in particular was in CRLF format whereas the others are in LF. On Visual Studio on Ubuntu for instance it created some problems, so i switched it to CRLF which is the default format for all of our OS

**Review by fcollot — APPROVED** [review]

**Review by Florent2305 — APPROVED** [review]
Ok

**Pull request merged**